### PR TITLE
feat: Support bitwarden cli

### PR DESCRIPTION
**Pull request by dploeger**

### CODE REVIEW

**Comment by dploeger** [comment]
This adds support for the bitwarden cli in CloudControl

**Comment by timdeluxe** [comment]
Sure you want to **assign** this to us? I think more you want us as reviewer, don't you?

File: README.md [hunk 1]
```diff
@@ -337,6 +339,15 @@ Installs [AzCopy](https://github.com/Azure/azure-storage-azcopy)
 * USE_azcopy: Enable this feature
 * DEBUG_azcopy: Debug this feature
 
+### bitwarden
+
+Installs the [Bitwarden cli](https://bitwarden.com/help/cli/)
```
Comment (timdeluxe) [inline README.md:344]: just cosmetics: i prefer cli in capital letters

File: feature/bitwarden/feature.yaml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+title: "🔐 Bitwarden"
+description: "Installs the [Bitwarden cli](https://bitwarden.com/help/cli/)"
```
Comment (timdeluxe) [inline feature/bitwarden/feature.yaml:2]: just cosmetics: i prefer cli in capital letters

File: feature/bitwarden/install.sh [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+  execHandle "Installing required libraries" sudo apk add gcompat
+fi
+
+TEMPDIR=$(mktemp -d)
+cd "${TEMPDIR}" || exit
+execHandle "Downloading bitwarden client" curl -L -o bw.zip "https://vault.bitwarden.com/download/?app=cli&platform=linux"
+execHandle "Unzipping bitwarden client" unzip bw.zip
+execHandle "Making bitwarden client executable" chmod +x bw
+execHandle "Installing bitwarden client" mv bw /home/cloudcontrol/bin
+rm -rf "${TEMPDIR}"
```
Comment (timdeluxe) [inline feature/bitwarden/install.sh:14]: Isn't the "cd - ..." missing at the end here?

**Review by timdeluxe — CHANGES_REQUESTED** [review]

**Comment by dploeger** [comment]
> Sure you want to **assign** this to us? I think more you want us as reviewer, don't you?

Right, sorry. 😄

File: README.md (hunk 1 above)
Comment (dploeger) [reply README.md]: Fixed

File: feature/bitwarden/feature.yaml (hunk 2 above)
Comment (dploeger) [reply feature/bitwarden/feature.yaml]: Fixed

File: feature/bitwarden/install.sh (hunk 3 above)
Comment (dploeger) [reply feature/bitwarden/install.sh:15]: Fixed

File: README.md [hunk 4]
```diff
@@ -19,6 +19,7 @@ Following features and tools are supported:
 
 * 🐟 Fish Shell
 * 📷 AzCopy
+* 🔐 Bitwarden
 * ⚙️  Direnv
```
Comment (ThomasMinor) [inline README.md:23]: Is there a superfluous blank before Direnv?

**Review by ThomasMinor — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (dploeger) [reply README.md]: Yes, you're right. Fixed.

File: README.md [hunk 5]
```diff
@@ -19,7 +19,8 @@ Following features and tools are supported:
 
 * 🐟 Fish Shell
 * 📷 AzCopy
-* ⚙️  Direnv
+* 🔐 Bitwarden
+* ⚙️ Direnv
 * ⛵️ Helm
 * 🛠  JQ
```
Comment (timdeluxe) [inline README.md:25]: well, if we look for superflous blanks: here is another one 😉

Comment (dploeger) [reply README.md:25]: Fixed

**Review by timdeluxe — APPROVED** [review]

**Review by ThomasMinor — APPROVED** [review]

**Pull request merged**